add_acronym_pronounce_to_json.py

```diff
@@ -0,0 +1,32 @@
+# Use csv of acronyms and translate, e.g. IBM -> I B M,
```

Comment:
Let's consolidate to camelCase or snake_case for the file names.

Reply:
changed to snake_case